modules/powermanagment/hddparm.nix

```diff
@@ -2,6 +2,6 @@
 
 {
   powerManagement.powerUpCommands = ''
-    ${pkgs.utillinux}/bin/for dev ${pkgs.utillinux}/bin/in $(${pkgs.utillinux}/bin/lsblk -d -o name,rota | ${pkgs.gnugrep}grep ' 1$' | ${pkgs.utillinux}/bin/awk '{print $1}'); ${pkgs.utillinux}/bin/do ${pkgs.hdparm} -S 240 -B 127 /dev/$dev; ${pkgs.utillinux}/bin/done
+    ${pkgs.utillinux}/bin/for dev ${pkgs.utillinux}/bin/in $(${pkgs.utillinux}/bin/lsblk -d -o name,rota | ${pkgs.gnugrep}grep ' 1$' | ${pkgs.utillinux}/bin/awk '{print $1}'); ${pkgs.utillinux}/bin/do ${pkgs.hdparm} -S 120 -B 127 /dev/$dev; ${pkgs.utillinux}/bin/done
```

Comment:
better to separate such small changes to different PRs so they can be review easier and accepted faster 
especially if they doesn't relate to anything else in that pr